src/support/ios.c

```diff
@@ -1074,6 +1076,17 @@ void ios_init_stdstreams(void)
     ios_stderr = (ios_t*)malloc_s(sizeof(ios_t));
     ios_fd(ios_stderr, STDERR_FILENO, 0, 0);
     ios_stderr->bm = bm_none;
+
+    // the following 'safe' variants must use `bm_line` / `bm_none`
```

Comment:
`bm_line` seems likely very, very unsafe, so only `bm_none` is okay here. Only when allocated as a local variable, then bm_line is good and works to make something like stderr much faster without sacrificing safety.

Reply:
Oh I understand your suggestion now. Happy to switch to the local variant with `bm_line` if you'd prefer (and will just change this to `bm_none` if not)

Reply:
I think this is fine, since changing to add the boilerplate everywhere seems unnecessary to add into this PR. If someone wants to do that later, they can and we'd remove `ios_safe_stderr` once done.